StortSpelProjekt/Engine/src/ECS/Components/TransformComponent.cpp

```diff
@@ -16,10 +16,15 @@ namespace component
 	{
 		m_pTransform = new Transform();
 
+		m_CameraLocked = false;
```

Comment:
TransformComponent Should not have a m_CameraLocked variable since it, more logical to move the camera to the entity instead of the entity to the camera (if there is no other way, fine).

Reply:
The camera is moved to the entity, not the other way around. This variable is needed to check if the camera is locked to the entity in order to be able to unsubscribe from the input events. If this was not there, the entity would still be moving even when the camera would not be locked